numpy/random/__init__.py

```diff
@@ -6,17 +6,15 @@
 ==================== =========================================================
 Utility functions
 ==============================================================================
-random               Uniformly distributed values of a given shape.
+random_sample        Uniformly distributed floats over ``[0, 1)``.
+random               Alias for `random_sample`.
 bytes                Uniformly distributed random bytes.
 random_integers      Uniformly distributed integers in a given range.
-random_sample        Uniformly distributed floats in a given range.
-random               Alias for random_sample
-ranf                 Alias for random_sample
-sample               Alias for random_sample
```

Comment:
I assume you removed this because it is not exported in `__all__`.

Reply:
Nope, I removed this simply because it was already absent from `random.__doc__`